backend/src/engine/oibus-engine.ts

```diff
@@ -163,51 +175,41 @@ export default class OIBusEngine extends BaseEngine {
       baseFolder,
       this.logger.child({ scopeType: 'north', scopeId: settings.id, scopeName: settings.name })
     );
-    if (north.isEnabled()) {
-      this.homeMetricsService.addNorth(north, settings.id);
-      // Do not await here, so it can start all connectors without blocking the thread
-      north.start().catch(error => {
-        this.logger.error(`Error while starting North connector "${settings.name}" of type "${settings.type}" (${settings.id}): ${error}`);
-      });
-    } else {
-      this.logger.trace(`North connector "${settings.name}" not enabled`);
-    }
     this.northConnectors.set(settings.id, north);
+    this.homeMetricsService.addNorth(north, north.getSettings().id);
   }
 
-  async addItemToSouth(southId: string, item: SouthConnectorItemDTO): Promise<void> {
-    await this.southConnectors.get(southId)?.addItem(item);
-  }
-
-  async deleteItemFromSouth(southId: string, item: SouthConnectorItemDTO): Promise<void> {
-    await this.southConnectors.get(southId)?.deleteItem(item);
-  }
-
-  async deleteAllItemsFromSouth(southId: string): Promise<void> {
-    await this.southConnectors.get(southId)?.deleteAllItems();
-  }
+  async startNorth(northId: string): Promise<void> {
+    const north = this.northConnectors.get(northId);
+    if (!north) {
+      this.logger.trace(`North connector ${northId} not set`);
+      return;
+    }
 
-  async updateItemInSouth(southId: string, oldItem: SouthConnectorItemDTO, newItem: SouthConnectorItemDTO): Promise<void> {
-    await this.southConnectors.get(southId)?.updateItem(oldItem, newItem);
+    // Do not await here, so it can start all connectors without blocking the thread
+    north.start().catch(error => {
+      this.logger.error(
+        `Error while starting North connector "${north.getSettings().name}" of type "${north.getSettings().type}" (${north.getSettings().id}): ${error}`
+      );
+    });
```

Comment:
This is called by the reload service here:
https://github.com/OptimistikSAS/OIBus/blob/650c83a3f23ccea1ca7d363525d373d6dc6a2a42/backend/src/service/reload.service.ts#L278-L281

Which sets the db status as `enabled` before the `this.oibusEngine.startNorth` call.
But if the `north.start()` call fails, the database will still have the `enabled` status set to true.

So the engine should handle this somehow, to have the `enabled` status set to false if the `north.start()` call fails.
This seems to be a bit hard to implement at the moment, because it would require to pass down some sort of `onErrorFn` function to the engine, which should come from the reload service.